shared/cfg/guest-os/Linux/Fedora/23.ppc64le.cfg

```diff
@@ -0,0 +1,19 @@
+- 23.ppc64le:
+    image_name = images/f23-ppc64le
+    vm_arch_name = ppc64le
+    os_variant = fedora23
+    no unattended_install..floppy_ks
+    boot_path = ppc/ppc64
+    unattended_file_kernel_param_name = ks
+    unattended_install, svirt_install:
+        kernel_params = 'repo=cdrom:/dev/disk/by-label/Fedora-Server-23-ppc64le'
```

Comment:
@ldoktor Do you get issues even with this config?, Can you paste your failure, which would be help me to understand more, Thanks in advance.